### ISSUE TITLE
Awesome Program! Some fixes...

### ISSUE THREAD

**Issue by Russell-Newton** [Latedi/DuckyAPI#1]
I looked through a lot of projects that claimed to be able to interface with Ducky RGB keyboards, and this was the first that worked for me.

In order to get this to work with the Ducky One 2 RGB, I used a different VID and PID, and I enumerated all of the devices to get to the one at the right usage_page. Additionally, I modified ColorSetter.cpp to have mappings that fit my US layout.

Great work!

If you're willing to make any changes, passing the VID, PID, and usage_table as command-line parameters for the .exe could be good. I have no idea how to do this, because I'm not proficient in C++ and building to .exe. :)

**Comment by Russell-Newton** [comment]
One other thing I noticed was that the program kept stopping up - the keyboard lighting would lock up, and nothing could control it until it was unplugged from the computer. I found out that the keyboard lighting controller's input buffer was locked up. This is because `ColorSetter::PushColors()` wasn't waiting for the keyboard to respond before sending another packet. a simple fix is to add the following code at the end of the method:
```cpp
uint32_t bytesReceived;
uint8_t *result = handler->Recv64(&bytesReceived);
```

I also found another packet protocol that requires less overall packets in order to set the colors.
